src/Symfony/Component/Cache/Tests/Adapter/TagAwareAndProxyAdapterIntegrationTest.php

```diff
@@ -47,11 +52,15 @@ public function testIntegrationUsingProxiedAdapterForTagsPool()
         $cache->invalidateTags(['baz']);
 
         $this->assertFalse($cache->getItem('foo')->isHit());
+
+        $cache->clear();
+        $proxiedAdapter->clear();
     }
 
     public function dataProvider(): array
     {
         return [
+            [new RedisAdapter(AbstractAdapter::createConnection('redis://'.getenv('REDIS_HOST')), str_replace('\\', '.', __CLASS__))],
```

Comment:
This should be skipped when the env var doesn't lead to a valid connection. See other test cases that use REDIS_HOST for inspiration.